scripts/testing.R

```diff
@@ -0,0 +1,29 @@
+# Script for testing functions
```

Comment:
Question for testing are you thinking about using this for the CI tests? Or is this kinda like a scratch pad that you are using while you are developing functions?

Reply:
So far, I've just been using this file like a scratch pad, but some stuff here might be useful for making CI tests.